src/event/handlers/change_events.js

```diff
@@ -19,6 +19,10 @@ var changeDoesNotBubble = (function() {
   // @license MIT
   var d = document.createElement('div');
   d.setAttribute('onchange', 't');
+  if (typeof d.attributes === 'undefined') {
+    // Element.attributes is undefined in an node environment
+    return;
```

Comment:
Doesn't really matter functionally, but can you have this return true so that changeDoesNotBubble stays a boolean